lib/rubycritic.rb

```diff
@@ -1,22 +1,7 @@
-require "rubycritic/configuration"
+require "rubycritic/command_factory"
 
 module Rubycritic
   def self.create(options = {})
```

Comment:
Why not go all the way and remove RubyCritic.create so that callers have to use RubyCritic::CommandFactory.create? 
That would be even more expressive.